viz/src/components/TheTape.vue

```diff
@@ -38,59 +57,46 @@ const addedBlankSpace = squaresAmountOnOneSideOfHead;
  * We add enough blank squares to all steps so that
  * there will always be the same amount of squares on the tape.
  */
-function fillStepsWithBlankSpacesOnSides(steps: TapeStep[]): TapeStep[] {
+function fillStepsWithBlankSpacesOnSides(steps: TapeStep[], blank: string): TapeStep[] {
   return steps.map(({ tape, ...props }) => ({
     tape: [
-      ...Array.from({ length: addedBlankSpace }).map(
-        () => blankCharacter.value
-      ),
+      ...Array.from({ length: addedBlankSpace }).map(() => blank),
```

Comment:
Thoughts: I have no issue with the related code `array.from` + `map` are widely known and readable, just FYI `array.from` can take a fill function
```
const array = Array.from({ length: 5 }, () => blank);
```